packages/eth/src/audius-token/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const AUDIUS_TOKEN_CONTRACT_ADDRESS =
```

Comment:
I can't recall if we have a staging token we need to support as well?

Reply:
we do - the prod values are just baked into the package for ease of use, but inside sdk we will provide stage config that overrides the default